web/templates/note/index.html.eex

```diff
@@ -0,0 +1,10 @@
+<h1>Listing Notes</h1>
+
+<table class="table">
```

Comment:
Why add a class? Also, a table seems like a weird use of html. Should this be a list?